Doc/Tutorial/chapter_entrez.tex

```diff
@@ -200,18 +200,20 @@ \section{ESearch: Searching the Entrez databases}
 
 As a final example, let's get a list of computational journal titles:
 \begin{verbatim}
->>> handle = Entrez.esearch(db="journals", term="computational")
+print("{} computational Journals found".format(record["Count"]))
```

Comment:
That should have three leading ``>>>`` signs and the expected result if you want it to be part of the "doctest".

However, since this appears later I think this is more likely an accident and this line should be removed?